src/components/proposals/proposal-deliverables.svelte

```diff
@@ -0,0 +1,99 @@
+<script lang="ts">
+  import clsx from 'clsx';
+  import { t } from '$lib/i18n';
+
+  import type { ProposalDeliverableStoryblok } from '$types/bloks';
+
+  export let data: ProposalDeliverableStoryblok[];
+
+  $: totalManpower = data.reduce((acc, entry) => (acc += +entry.manpower), 0);
+</script>
+
+<div class="overflow-x-scroll mt-10 md:mt-14 lg:mt-20">
+  <div class="border-b border-foreground-secondary min-w-[780px]">
+    <div
+      class={clsx(
+        'container mx-auto px-container py-2',
+        'grid grid-cols-[1fr_2fr_1fr] gap-10 md:gap-12'
+      )}
+    >
+      <p class="text-[0.69rem]/[0.88rem] uppercase tracking-wider text-foreground-secondary">
```

Comment:
There's a lot of repetition of this text style `text-[0.69rem]/[0.88rem] tracking-wider`, that I feel could live in `tailwind.config` on the `extend.fontSize` property to be easily maintainable.